raiden/api/objects.py

```diff
@@ -18,36 +18,63 @@ class ChannelList(FlatList):
     pass
 
 
+class TokensList(FlatList):
+    pass
+
+
+class PartnersPerTokenList(FlatList):
+    pass
+
+
 class EventsList(FlatList):
     pass
 
 
+class Token(object):
+    def __init__(self, token_address):
+        self.address = token_address
+
+
+class PartnersPerToken(object):
+    def __init__(self, partner_address, channel):
+        self.partner_address = partner_address
+        self.channel = channel
+
+
 class Channel(object):
-    def __init__(self, channel_address, asset_address, partner_address, settle_timeout, reveal_timeout, deposit, status):
+    def __init__(
+            self,
+            channel_address,
+            token_address,
+            partner_address,
+            settle_timeout,
+            reveal_timeout,
+            balance,
+            state):
         self.channel_address = channel_address
-        self.asset_address = asset_address
+        self.token_address = token_address
         self.partner_address = partner_address
         self.settle_timeout = settle_timeout
         self.reveal_timeout = reveal_timeout
-        self.deposit = deposit
-        self.status = status
+        self.balance = balance
+        self.state = state
 
 
 class TransferReceived(object):
```

Comment:
This feels like a misnomer.

- It's not clear the intended use, it could be for a token transfer that completed and changed the local node's balance or for a protocol message received.
- If it's for a protocol message it doesn't distinguish among the existing types (direct, mediated, refunds).

On a side note: Perhaps we need a nomenclature to distinguish among these, perhaps payments vs. transfer ?